pkgs/development/python-modules/django-rest-auth/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, requests
+, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-rest-auth";
+  version = "0.9.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ad155a0ed1061b32e3e46c9b25686e397644fd6acfd35d5c03bc6b9d2fc6c82a";
+  };
+
+  doCheck = true;
+
+  propagatedBuildInputs = [ django requests django-rest-auth ];
+
+  buildInputs = [ django requests ];
+
+  meta = with stdenv.lib; {
+    description = ''
+      This app makes it extremely easy to build Django powered SPA's (Single
+      Page App) or Mobile apps exposing all registration and authentication
+      related functionality as CBV's (Class Base View) and REST (JSON)
+      http://www.tivix.com
```

Comment:
That's way too long but may be used for `longDescription` if you wish to have one.